app/main/views.py

```diff
@@ -355,7 +355,117 @@ def divisions():
     return render_template('divisions.html')
 
 
-@main.route('/it-support', methods=['GET', 'POST'])
+@main.route('/divisions/administration', methods=['GET'])
```

Comment:
For all of these routes, can't we just have a single route that has a string as it's last element and have that query the database and render the page?

e.g.
```python
@main.route('/divisions/<division_name>', methods=['GET'])
def division_pages(division_name):
    users = get_users_by_division(division_name)
    rooms = get_rooms_by_division(division_name)
    return render_template('divisions/{division_name}.html'.format(division_name=division_name), users=users, rooms=rooms)
```